### PR TITLE
Add a model Mentor details 

### PR DESCRIPTION
**Pull request by Winchestery**
Added a  request to mentor details feature, using GET mentor by Id request

### CODE REVIEW

File: src/features/index.js [hunk 1]
```diff
@@ -18,7 +18,6 @@ export * from './list-of-mentors';
 export * from './edit-students-details';
 export * from './edit-group';
 export * from './edit-lesson';
-export * from './edit-mentor';
```
Comment (MozdolevskyiMaksym) [inline src/features/index.js:21]: don`t we need this export?

File: src/app.js [hunk 2]
```diff
@@ -1,12 +1,13 @@
...
-import { Routes } from '@features/index.js';
+import { Counter, Routes } from './features/index.js';
 import Icon from './icon.js';
 
 export const App = () => (
   <>
     <h1>Hello, I`m App Component!</h1>
     <Icon icon="Plus" size={32} className="icon" color="#FFB800" />
     <Icon icon="Edit" viewBox="0 0 50 50" color="#FFB800" />
+    <Counter />
```
Comment (MozdolevskyiMaksym) [inline src/app.js:10]: Please, remove counter component from here

File: src/features/list-of-mentors/list-of-mentors.js [hunk 3]
```diff
@@ -2,91 +2,91 @@ import React, { useState, useEffect } from 'react';
 import { shallowEqual, useSelector } from 'react-redux';
 import { useHistory } from 'react-router-dom';
 import classNames from 'classnames';
-import { Card, Search, Button, WithLoading } from '../../components/index.js';
-import Icon from '../../icon.js';
-import styles from './list-of-mentors.scss';
 import { useActions } from '@/shared/index.js';
```
Comment (MozdolevskyiMaksym) [inline src/features/list-of-mentors/list-of-mentors.js:5]: try to sort all imports

File: src/features/mentor-details/mentor-details.js [hunk 4]
```diff
@@ -0,0 +1,59 @@
...
+
+  const history = useHistory();
+
+  useEffect(() => {
+    dispatchLoadMentors(id);
+  }, [dispatchLoadMentors, id]);
+
+  if (error) {
+    history.push('/404');
+  }
```
Comment (MozdolevskyiMaksym) [inline src/features/mentor-details/mentor-details.js:31]: I guess it should be like that
```
useEffect(() => {
  if (error) {
    history.push('/404');
  }
}, [error]);
```

File: src/app.js (hunk 2 above)
Comment (Winchestery) [reply src/app.js:10]: done

File: src/features/mentor-details/mentor-details.js (hunk 4 above)
Comment (Winchestery) [reply src/features/mentor-details/mentor-details.js:31]: done

**Review by sarhan-azizov — APPROVED** [review]

**Pull request merged**